src/Routes/ImageManagerDetail/constants.js

```diff
@@ -48,13 +48,11 @@ export const imageStatusMapper = {
 };
 
 export const distributionMapper = {
-  'rhel-84': 'RHEL 8.4',
   'rhel-85': 'RHEL 8.5',
 };
 
 export const releaseMapper = {
   'rhel-85': 'Red Hat Enterprise Linux (RHEL) 8.5',
-  'rhel-84': 'Red Hat Enterprise Linux (RHEL) 8.4',
```

Comment:
We need a way to avoid breaking the update image wizard, I suggest we disable the update button if an image is an RHEL 8.4 image.

Reply:
We need a way to stop creating new images with rhel 8.4, the best way is remove them completely from the wizard. 

remove updating when 8.4 is a feature that is not assumed by the new deviceview, and a story to be opened and may take time to relise on the backed. 

Why removing from the wizard is a problem ?

Reply:
I think the best solution is to hard core 8.5 to the create image wizard so we don't break the update image wizard with 8.4 images.

Reply:
That's the purpose of this PR, we are enforcing rhel 8.5 on create wizard.

Reply:
![image](https://user-images.githubusercontent.com/131553/165375370-fff45676-924d-45eb-acfd-f2d4e7d0896b.png)

Reply:
This PR will remove support for the update image wizard. We need to remove 8.4 from only the create image wizard but not from the update image wizard.